src/server/handlers/github-auth.js

```diff
@@ -116,6 +108,15 @@ export const verify = (req, res, next) => {
       }
     });
 
+    // Save auth token and user info to session
+    req.session.githubAuthenticated = true;
+    req.session.token = body.access_token;
+    req.session.user = {
+      ...userResponse.body,
+      orgs,
+      hasAddedSnaps
```

Comment:
Exposing `hasAddedSnaps` in session user object (which is passed to initial redux store state).